src/libs/actions/Wallet.ts

```diff
@@ -0,0 +1,420 @@
+                isPendingOnfidoResult: null,
+                shouldShowFailedKYC: true,
+            },
+        },
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.WALLET_TERMS,
+            value: {
+                isLoading: false,
+            },
+        },
+    ];
+
+    type AcceptWalletTermsParams = {
+        hasAcceptedTerms: boolean;
+        reportID: number;
+    };
+
+    const requestParams: AcceptWalletTermsParams = {hasAcceptedTerms: parameters.hasAcceptedTerms, reportID: parameters.chatReportID};
+
+    API.write('AcceptWalletTerms', requestParams, {optimisticData, successData, failureData});
+}
+
+/**
+ * Fetches data when the user opens the InitialSettingsPage
+ */
+function openInitialSettingsPage() {
+    API.read('OpenInitialSettingsPage', {});
+}
+
+/**
+ * Fetches data when the user opens the EnablePaymentsPage
+ */
+function openEnablePaymentsPage() {
+    API.read('OpenEnablePaymentsPage', {});
+}
+
+function updateCurrentStep(currentStep: ValueOf<typeof CONST.WALLET.STEP>) {
+    Onyx.merge(ONYXKEYS.USER_WALLET, {currentStep});
+}
+
+function answerQuestionsForWallet(answers: WalletQuestion[], idNumber: string) {
+    const idologyAnswers = JSON.stringify(answers);
+
+    const optimisticData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.WALLET_ADDITIONAL_DETAILS,
+            value: {
+                isLoading: true,
+            },
+        },
+    ];
+
+    const successData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.WALLET_ADDITIONAL_DETAILS,
+            value: {
+                isLoading: false,
+            },
+        },
+    ];
+
+    const failureData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.WALLET_ADDITIONAL_DETAILS,
+            value: {
+                isLoading: false,
+            },
+        },
+    ];
+
+    type AnswerQuestionsForWallet = {
+        idologyAnswers: string;
+        idNumber: string;
+    };
+
+    const requestParams: AnswerQuestionsForWallet = {
+        idologyAnswers,
+        idNumber,
+    };
+
+    API.write('AnswerQuestionsForWallet', requestParams, {
+        optimisticData,
+        successData,
+        failureData,
+    });
+}
+
+function requestPhysicalExpensifyCard(cardID: number, authToken: string, privatePersonalDetails: PrivatePersonalDetails) {
+    const {
+        legalFirstName,
+        legalLastName,
+        phoneNumber,
+        address: {city, country, state, street, zip},
+    } = privatePersonalDetails;
+
+    type RequestPhysicalExpensifyCardParams = {
+        authToken: string;
+        legalFirstName: string;
+        legalLastName: string;
+        phoneNumber: string;
+        addressCity: string;
+        addressCountry: string;
+        addressState: string;
+        addressStreet: string;
+        addressZip: string;
+    };
```

Comment:
```suggestion
```

No type is needed here

Reply:
Same as before - The convention we try to follow is to have every API.write call's parameters strictly typed to ensure type safety.